x-pack/test/functional/apps/endpoint/index.ts

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+export default ({ loadTestFile }: FtrProviderContext) => {
+  describe('Endpoint app', function() {
+    // this.tags('ciGroup7');
```

Comment:
I think you'll want to keep this, if tests aren't assigned a ciGroup, CI will fail and require a ciGroup be assigned.